### DISCUSSION TITLE
automated invoices

### DISCUSSION THREAD

**Discussion by chollan** [kimai/kimai#3157] [Questions & Answers] [answered]
is there a plugin or another way to automate invoice creation and delivery?

i'm a symfony developer so i can look into adding this in my environment, i just wanted to ask if there was such a thing before i started my own development.

**Comment by kevinpapst** [comment, marked as answer]
Hi Curtis, there is a command that was made for this purpose: 
https://www.kimai.org/documentation/invoices.html#create-invoices-with-cronjobs

**Comment by chollan** [comment]
Awesome!  thank you.  another question - i want to automate the sending of invoices to my clients.  is that somthing that exists already or is that also custom development?

**Reply by kevinpapst** [reply]
Planned for the future, but not an existing feature right now.

**Reply by chollan** [reply]
If i were to write this into my instance, would you be interested in a PR for this feature?

**Reply by kevinpapst** [reply]
Yes sure! Question is: what about the email template? There should be a default text, which should be editable as well.

**Reply by chollan** [reply]
Not really sure yet.  i'll definitely keep that in mind when i start development into this feature.

**Reply by jens-2305** [reply]
Hi, is there allready  an solution for the automatic sending for the invoices ?
